.gitignore

```diff
@@ -4,3 +4,6 @@ data/etfs/
 data/stocks/
```

Comment:
Ignore all data directories with `data/`
Update the tracked files with `git rm --cached -r data/`